schema.sql

```diff
@@ -2,4 +2,5 @@
 
 
 CREATE TABLE animals (id integer, name text, date_of_birth date, escape_attempts integer, neutered boolean, weight_kg decimal) ;
+ALTER TABLE animals ADD species varchar
```

Comment:
- [ ] [optional] It would have been nice to set the  varchar limit that you want for this column eg `ALTER TABLE animals ADD species varchar(255);`